packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -61,38 +62,45 @@ export const sliderStyles: (
         height: 100%;
         grid-template-columns: calc(var(--thumb-size) * 1px) 1fr;
     }
-
-    :host(:${focusVisible}) .thumb-cursor {
-        box-shadow: 0 0 0 2px ${fillColor}, 0 0 0 4px ${focusStrokeOuter};
-    }
-
     .thumb-container {
         position: absolute;
         height: calc(var(--thumb-size) * 1px);
         width: calc(var(--thumb-size) * 1px);
         transition: all 0.2s ease;
-        color: ${neutralForegroundRest};
-        fill: currentcolor;
     }
     .thumb-cursor {
+        display: flex;
+        position: relative;
         border: none;
         width: calc(var(--thumb-size) * 1px);
         height: calc(var(--thumb-size) * 1px);
-        background: ${neutralForegroundRest};
-        border-radius: calc(${controlCornerRadius} * 1px);
+        background: ${fillColor};
+        border: calc(${focusStrokeWidth} * 1px) solid ${strokeControlStrongRest};
```

Comment:
```suggestion
        border: calc(${strokeWidth} * 1px) solid ${strokeControlStrongRest};
```
We should keep semantic names for their purpose. Let's align the design here with checkbox and radio.